src/main/scala/firrtl/passes/formal/AssertSubmoduleAssumptions.scala

```diff
@@ -0,0 +1,33 @@
+
+package firrtl.passes.formal
+
+import firrtl.passes.{Pass, PullMuxes, ReplaceAccesses, ResolveFlows}
+import firrtl.ir.{Assert, Assume, Circuit, Statement}
+import firrtl.options.Dependency
+import firrtl.stage.TransformManager.TransformDependency
+import firrtl.{CircuitState, FirrtlUserException, Transform}
+
+
+object AssertSubmoduleAssumptions extends Pass {
```

Comment:
Annotation to enable/disable this pass
See NoDCE annotation

What should the default be? Solicit feedback from formal people

Non-blocking

Reply:
Simulation tools automatically treat assume as asserts.  So this pass may not be necessary.